src/Controls/src/Core/HandlerImpl/Button/Button.cs

```diff
@@ -15,7 +15,7 @@ public partial class Button
 		public static IPropertyMapper<IButton, ButtonHandler> ControlsButtonMapper = new PropertyMapper<Button, ButtonHandler>(ButtonHandler.Mapper)
 		{
 			[nameof(ContentLayout)] = MapContentLayout,
-#if IOS
+#if __IOS__
```

Comment:
What is the reason for this one? Is there a difference?

Reply:
reverted